package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@webex/common": "^1.105.0",
-    "@webex/component-adapter-interfaces": "^1.11.0"
+    "@webex/component-adapter-interfaces": "^1.13.0"
```

Comment:
Things move fast around here! I think latest is now `1.16.0`. Let's use latest so we don't have to update again later. Also, please use a separate `fix` commit for any changes related to dependencies

Reply:
@lalli-flores do you mean just updating the version to `1.16.0` or using the `latest` tag  e.g: `"@webex/component-adapter-interfaces": "latest"` ?